index.html

```diff
@@ -103,148 +112,65 @@ <h1>Main Program</h1>
               <img src="./images/lecture.png" alt="fashionicons" />
               <h4>Lecture</h4>
               <p>
-                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dicta
-                adipisci ipsa rerum tempore voluptate harum aspernatur dolorum
-                alias repellat assumenda?
+                We teach you, what it takes to know about fashion and be
+                fashionable. You can proudly pick colors and trend
               </p>
             </li>
             <li>
               <img src="./images/face.png" alt="fashionicons" />
               <h4>Men Fashion Exhibition</h4>
               <p>
-                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dicta
-                adipisci ipsa rerum tempore voluptate harum aspernatur dolorum
-                alias repellat assumenda?
+                Men fashin is its own unique style, Talks on cloths that brings
+                the calm and bold you.
               </p>
             </li>
```

Comment:
Hello @Ginohmk, good job so far!, I am suggesting that these `li`s are instead wrapped in `article` tags. This would make your code more readable and professional

For example you could have 👇🏻 
```
<section class="program-list">
   <article></article>
   <article></article>
   <article></article>
   <article></article>
   <article></article>
</section>
```

Reply:
Is this optional too ?, Please i do not understand your comment on my code. the code you reference is outdated as it has lorem text

Reply:
Hi @Ginohmk, I am sorry for not specifying it earlier but it is a required change. Thank you!

Reply:
Alright, thank you. I will work on it now. thank you for explaining better to me.